src/main/java/com/work/management/dto/BulkEmployeeDtoResponse.java

```diff
@@ -0,0 +1,20 @@
+package com.work.management.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.work.management.web.rest.resource.EmployeeResource;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class BulkEmployeeDtoResponse {
+
+  private List<EmployeeResource> employeeResourceList;
+
```

Comment:
Don;t appned list in the name of variable, it is evident from datatype, rename it to emplyeeResources